README.markdown

```diff
@@ -1,6 +1,6 @@
 # DistrictBuilder
 
-[![Build Status](https://travis-ci.org/PublicMapping/DistrictBuilder.svg?branch=develop)](https://travis-ci.org/PublicMapping/DistrictBuilder)
+[![Build Status](https://travis-ci.org/PublicMapping/DistrictBuilder.svg?branch=develop)](https://travis-ci.org/azavea/district-builder-dtl-pa)
```

Comment:
I think that we want to keep the previous state here.

Reply:
This change should be the other way around. I think there's something wrong with the version history.

Reply:
If I pull down the branch locally, it contains the DTL repository name:

```
$ head README.markdown
# DistrictBuilder

[![Build Status](https://travis-ci.org/PublicMapping/DistrictBuilder.svg?branch=develop)](https://travis-ci.org/azavea/district-builder-dtl-pa)

[DistrictBuilder](http://www.districtbuilder.org/) is software created by the [Public Mapping Project](http://www.publicmapping.org/resources/software).

The development environment is docker-compose for services inside a Vagrant virtual machine.

## Table of Contents
  * [Installation](#installation)
```